src/geo/ui/infowindow.js

```diff
@@ -327,108 +329,126 @@ var Infowindow = View.extend({
     return !!this.$('.js-infowindow').attr('data-cover');
   },
 
+  _containsTemplateCover: function () {
+    return this.$('.js-cover img').length > 0;
+  },
+
   /**
    *  Get cover URL
    */
   _getCoverURL: function () {
     var content = this.model.get('content');
 
-    if (content && content.fields && content.fields.length > 0) {
+    if (this.$('.js-cover img').attr('src')) {
```

Comment:
_Totally optional: I'd create a var storing the src of the img, being used twice._